src/REPLMode/REPLMode.jl

```diff
@@ -365,7 +365,7 @@ function Command(statement::Statement)::Command
     # arguments
     arg_spec = statement.spec.argument_spec
     arguments = arg_spec.parser(statement.arguments, options)
```

Comment:
`default_parser` looks like it always returns `Vector{String}` but I don't know enough to promise that's true for all. `Command` (which gets called with `arguments`) has that field typed as `::Vector` with no specified eltype. Typing it `::AbstractVector` might be safe in general? I'd defer to others more knowledgeable about Pkg internals than I.

WRT `Command`, it might be a good idea to modify that field type declaration to something concrete.